Diagnostics/Get-NetView.PS1

```diff
@@ -1722,7 +1723,7 @@ function Worker {
 
     EnvSetup          -OutDir $workDir
 
-    Version           -OutDir $workDir
+    Metadata          -OutDir $workDir -Params $PSBoundParameters.GetEnumerator()
```

Comment:
It isn't possible to pass `$PSBoundParameters` directly because it is an automatic variable. AFAIK this is the best way to do this.